private/server/utils/data.ts

```diff
@@ -51,7 +71,13 @@ export const getRoninOptions = (
     return res;
   };
 
-  const dataFetcherWaitUntil = c.executionCtx.waitUntil?.bind(c.executionCtx);
+  let dataFetcherWaitUntil: ExecutionContext['waitUntil'] = EXECUTION_CONTEXT.waitUntil;
+  try {
+    dataFetcherWaitUntil = c.executionCtx.waitUntil?.bind(c.executionCtx);
+  } catch (_err) {
+    if (VERBOSE_LOGGING)
+      console.warn('No execution context provided, using default implementation.');
+  }
```

Comment:
This assignment could fail because when using `@hono/node-server` no execution context is provided & so an error will be thrown. But it's fine if it does because we have the mock implementation as a default.
See: https://github.com/honojs/hono/blob/ebc7e4bf4f9546404b4d436dd673069c432e71ca/src/context.ts#L384-L396